dev-packages/e2e-tests/test-applications/angular-17/package.json

```diff
@@ -40,11 +41,18 @@
     "karma-coverage": "~2.2.0",
     "karma-jasmine": "~5.1.0",
     "karma-jasmine-html-reporter": "~2.1.0",
-    "typescript": "~5.3.2",
     "ts-node": "10.9.1",
+    "typescript": "~5.3.2",
     "wait-port": "1.0.4"
   },
   "volta": {
     "extends": "../../package.json"
+  },
+  "pnpm": {
+    "overrides": {
+      "@sentry/browser": "latest || *",
+      "@sentry/core": "latest || *",
+      "@sentry/utils": "latest || *"
+    }
```

Comment:
Noticed that this seems to be required to ensure the local/verdaccio version of the dependencies of `@sentry/angular-ivy` are taken instead of the ones uploaded to NPM.